ig/local-generate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+docker build -t ig-local-dev -f Dockerfile-local-dev .
+docker run -it --rm --mount type=bind,source="$(pwd)",target=/trusted-intermediary ig-local-dev
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
```suggestion
if [[ "$OSTYPE" == "darwin"* ]] || [[ "$OSTYPE" == "linux-gnu" ]]; then
```
This will open the browser for linux users

Reply:
Does Linux have the `open` command?  I thought it didn't.  According to the man page for `open`, it came from NextStep which is the precursor to Mac OS X (10).

Reply:
It does. I think the problem I had was that the conditional statement, once I added the the rest of the code, it opened a browser for me.

Reply:
Oh, super cool!